api-doc/openapi.yaml

```diff
@@ -1,48 +1,514 @@
 openapi: 3.0.3
 info:
-  version: 0.0.1
+  version: 0.0.3.
   title: Vauhtijuoksu API
 paths:
-  /target:
+  /donations:
     get:
       responses:
-        200:
-          $ref: '#/components/responses/TargetOk'
+        '200':
+          $ref: '#/components/responses/DonationsAllOk'
+  /donation/{donationsId}:
+    parameters:
+      - name: donationsId
+        in: path
+        required: true
+        description: Donation id (uuid).
+        schema:
+          type: integer
+          format: int64
+          minimum: 1
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/DonationsOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Donations'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+  /gamedata:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/GameDataAllOk'
+  /gamedata/{gamedataId}:
+    parameters:
+      - name: gamedataId
+        in: path
+        required: true
+        description: Game data id (uuid).
+        schema:
+          type: integer
+          format: int64
+          minimum: 1
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/GameDataOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GameData'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+  /stream_metadata:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/StreamMetadataOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StreamMetadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
```

Comment:
What's the point here? Indicate that stream has ended?

Reply:
There is no point for delete endpoint for this, so removing